tdrs-backend/tdpservice/data_files/serializers.py

```diff
@@ -38,9 +40,15 @@ class Meta:
             "section",
             "created_at",
             "ssp",
-            "submitted_by"
+            "submitted_by",
+            "has_error"
         ]
 
+    def get_has_error(self, obj):
+        """Return whether the file has an error."""
+        parser_errors = ParserError.objects.filter(file=obj.id)
+        return len(parser_errors) > 0
```

Comment:
i think `return ParserError.objects.filter(file=obj.id).count()` may potentially be more efficient by offloading the work to the database